Update Deployment URL Retrieval Procedure in Airflow REST API (#1786)
astronomer/docs: astro/airflow-api.md

:::info

The token is only valid for 24 hours. If you need to call the Airflow API only once, you can retrieve a single 24-hour access token at `https://cloud.astronomer.io/token` in the Cloud UI.

If you've configured a [CI/CD process](ci-cd.md) and you want to avoid generating an access token manually, Astronomer recommends that you automate the API request to generate a new access token.

:::

### Retrieve the Deployment URL

> In the Astro CLI, run the following command:

Reviewer: ```suggestion
Run the following command to retrieve a Deployment URL:
```
To provide context you could say "in your terminal", but you're not "in the Astro CLI" when you're running CLI commands.
